Translation corrections (#165)
07th-mod/higurashi-console-arcs: Update/some_omake.txt

			NULL, " Anyway, let's get started.", Line_WaitForInput);
	ModPlayVoiceLS(4, 38, "ps3/s20/38/vtp_tamako1045", 256, TRUE);
	OutputLine(NULL, "これから『ひぐらし』のアドバンスドストーリーを一緒に進めていく仲間としての最初の親睦ってことで、ね？」",

> 			NULL, " This will be our first bonding sessions as the cast of these side stories \"Higurashi\", won't it?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " This will be our first bonding session as the cast of the \"Higurashi\" Advanced Story series, won't it?\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s20/28/vtp_tomoe1038", 256, TRUE);
	OutputLine(NULL, "「えーっとですね。",
			NULL, "\"Well, you see.", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s20/28/vtp_tomoe1039", 256, TRUE);
	OutputLine(NULL, "実は私、元々オカルト的なものってあまり信じないほうなんですが、今回の一連は特に自信を持って『祟り』ってのは何かのトリックだと思っています」",

> 			NULL, " Actually, I didn't really believe there was supernatural phenomena to begin with, but the incidents of this case in particular have given me confidence that there must be some trick to the \"curse\".\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Actually, I didn't really believe in supernatural phenomena to begin with, but the incidents of this case in particular have given me confidence that there must be some trick to the \"curse\".\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 38, "ps3/s20/38/vtp_tamako1052", 256, TRUE);
	OutputLine(NULL, "「う～ん……。",
			NULL, "\"Hmm......", Line_WaitForInput);
	ModPlayVoiceLS(4, 38, "ps3/s20/38/vtp_tamako1053", 256, TRUE);
	OutputLine(NULL, "安易にオカルト的な見解でとらえるのは何だかしゃくだけど、どう考えても超常的なものが暗躍しているとしか思えないんだけどなぁ」",

> 			NULL, " It does seem a bit of a cop-out to simply accept the supernatural explanation, but no matter how I consider it, the only thing that makes sense is that something supernatural is going on behind the scenes.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " It does seem a bit of a cop-out to simply accept the supernatural explanation, but no matter how I think about it, the only thing that makes sense is that something supernatural is going on behind the scenes.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) {
		OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_Continue);
		OutputLine(NULL, "「最初の事件では、犠牲者が出たことで村の人たちが『祟り』の存在を強く感じて、ますます『オヤシロさま』への信奉を篤くしていったよね？」",

> 				NULL, "<size=-2>\"In the first incident, the discovery of the victims made more people believe in to the \"curse\" and deepened their faith in \"Oyashiro-sama\", didn't it?\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "<size=-2>\"In the first incident, the discovery of the victims made more people believe in the \"curse\" and deepened their faith in \"Oyashiro-sama\", didn't it?\"", Line_Normal);
```

	} else {
		OutputLine(NULL, "「最初の事件では、犠牲者が出たことで村の人たちが『祟り』の存在を強く感じて、ますます『オヤシロさま』への信奉を篤くしていったよね？」",

> 				NULL, "\"In the first incident, the discovery of the victims made more people believe in to the \"curse\" and deepened their faith in \"Oyashiro-sama\", didn't it?\\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "\"In the first incident, the discovery of the victims made more people believe in the \"curse\" and deepened their faith in \"Oyashiro-sama\", didn't it?\\"", Line_Normal);
```

	}
	ClearMessage();

//夏美rvS20/36/VTP_natsumi1032_1.「……でも、二度目はその存在を一番敬う立場にいるはずの、オヤシロさまの巫女を殺している。kvS20/36/VTP_natsumi1033.これって、自分を信奉させるためって目的を考えたら、絶対に逆効果だよね？」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 36, "ps3/s20/36/vtp_natsumi1032_1", 256, TRUE);
	OutputLine(NULL, "「……でも、二度目はその存在を一番敬う立場にいるはずの、オヤシロさまの巫女を殺している。",

> 			NULL, "\"......But in the second incident was the murder of Oyashiro-sama's shrine maiden, who should have been the person most respected by Oyashiro-sama. ", GetGlobalFlag(GLinemodeSp));

Reviewer: ```suggestion
			NULL, "\"......But the second incident was the murder of Oyashiro-sama's shrine maiden, who should have been the person most respected by Oyashiro-sama. ", GetGlobalFlag(GLinemodeSp));
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c8647c>暁</color>", NULL, "<color=#c8647c>Akira</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 39, "ps3/s20/39/vtp_akira1024", 256, TRUE);
	OutputLine(NULL, "「確かに、……言われてみれば変だな。",

> 			NULL, "\"Yeah... now that mention it, it is strange.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Yeah... now that you mention it, it is strange.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c8647c>暁</color>", NULL, "<color=#c8647c>Akira</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 39, "ps3/s20/39/vtp_akira1027", 256, TRUE);
	OutputLine(NULL, "信じている人もいない人も惨劇に巻き込まれるようになったら、逆に『オヤシロさま』に関わることこそが危険なことだと皆が考え出す」",
			NULL, "Believers and nonbelievers were both caught up in the tragedy, which makes everyone start to think that getting involved with \"Oyashiro-sama\" is dangerous.\"", Line_Normal);
	ClearMessage();

//暁rvS20/39/VTP_akira1028.「ということは、『オヤシロさま』は自分の信者を自分自身で離れさせようとしている？kvS20/39/VTP_akira1029.　……どう考えても、それはおかしいよな」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c8647c>暁</color>", NULL, "<color=#c8647c>Akira</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 39, "ps3/s20/39/vtp_akira1028", 256, TRUE);
	OutputLine(NULL, "「ということは、『オヤシロさま』は自分の信者を自分自身で離れさせようとしている？",

> 			NULL, "\"Which would mean that \"Oyashiro-sama\" would be trying to push his own followers away?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Which would mean that \"Oyashiro-sama\" is trying to push his own followers away?", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s20/37/vtp_chisato1072", 256, TRUE);
	OutputLine(NULL, "「…………つまり、南井さんの考えだとこの『祟り』は、『オヤシロさま』の意思によるものではない、ということですか？」",

> 			NULL, "\".........In other word, Minai-san's assertion is that the \"curse\" is not caused by \"Oyashiro-sama's\" will?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\".........In other words, Minai-san's assertion is that the \"curse\" is not caused by \"Oyashiro-sama's\" will?\"", Line_Normal);
```

			NULL, "\"Good luck, Minai-san!", Line_WaitForInput);
	ModPlayVoiceLS(4, 36, "ps3/s20/36/vtp_natsumi1037", 256, TRUE);
	OutputLine(NULL, "　次回も頼もしい活躍、期待していますから！」",

> 			NULL, " I'm looking forward to see you play an active and reliable role next time!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I'm looking forward to seeing you play an active and reliable role next time!\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s20/37/vtp_chisato1077", 256, TRUE);
	OutputLine(NULL, "「いや～、今回いいこと聞いちゃったわ。",
			NULL, "\"Man~, now isn't that a juicy little story.", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s20/37/vtp_chisato1078", 256, TRUE);
	OutputLine(NULL, "このネタは、まだしばらく使えそうだね♪」",

> 			NULL, " I'll be able to make use of itfor a while♪.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I'll be able to make use of it for a while♪.\"", Line_Normal);
```
